### DISCUSSION TITLE
Documentation or examples on how to use argocd go-packages

### DISCUSSION THREAD

**Discussion by rdramana85** [argoproj/argo-cd#11320] [Q&A]
Hi - we want to use argocd go-packages to perform the tasks usually done with argocd-cli. Eg : "argocd app create". Can you provide documentation or examples on how to use go-packages?

**Comment by crenshaw-dev** [comment]
Here's the basic info for importing the package: https://argo-cd.readthedocs.io/en/stable/user-guide/import/

For examples, I'd recommend just looking through the [CLI code](https://github.com/argoproj/argo-cd/blob/master/cmd/argocd/commands/app.go).
